packages/oui-header-tabs/src/header-tab-content-transclude.directive.js

```diff
@@ -0,0 +1,27 @@
+export default () => {
+    "ngInject";
+    return {
+        restrict: "A",
+        require: "^ouiHeaderTabs",
+        link: (scope, elm, attrs) => {
+            const tab = scope.$eval(attrs.ouiTabContentTransclude);
+            tab.$transcludeFn((contents) => {
```

Comment:
Oww. I think we have to sit down all together and discuss about the tabs before simply copy paste... the past and doing weird things.